scripts/validate_docstrings.py

```diff
@@ -44,86 +44,183 @@
 PRIVATE_CLASSES = ['NDFrame', 'IndexOpsMixin']
 
 
-def _load_obj(obj_name):
-    for maxsplit in range(1, obj_name.count('.') + 1):
-        # TODO when py3 only replace by: module, *func_parts = ...
-        func_name_split = obj_name.rsplit('.', maxsplit)
-        module = func_name_split[0]
-        func_parts = func_name_split[1:]
-        try:
-            obj = importlib.import_module(module)
-        except ImportError:
-            pass
-        else:
-            continue
+def get_api_items():
```

Comment:
Thanks for the docstring it helps a lot! Is there a way to make a minimal test for this? Don't want to go overboard with tests here but I am worried that if this were to somehow break in the future it could allow subtle failures in subsequent tests to go through unnoticed

Reply:
Makes sense to me, but I'd address it in a separate PR